provisioning/service/samples/create_individual_x509_enrollment

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+'use strict';
+var fs = require('fs');
+var provisioningServiceClient = require('azure-iot-provisioning-service').ProvisioningServiceClient;
+
+var deviceID = process.argv[2];
+var connectionString = process.argv[3];
+var serviceClient = provisioningServiceClient.fromConnectionString(connectionString);
+
+var certFile = __dirname + "\\cert\\" + deviceID + "-cert.pem";
```

Comment:
>__dirname + "\\cert\\" + deviceID + "-cert.pem"; [](start = 15, length = 48)

same as #196, using path.join or path.resolve will guarantee cross-platform functionality

Reply:
changed this